src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -348,6 +356,8 @@ BaseWSPeer<Handler, Impl>::on_write_fin(error_code const& ec)
         return fail(ec, "write_fin");
     wq_.pop_front();
     if (do_close_)
+    {
```

Comment:
Should `do_close_` be reset to `false` here? What if `on_write_fin` is called again?

Reply:
Thanks for the feedback. Coincidentally, I had the realization about not needing `closing_` while having dinner, and came in to fix it before I even saw your comments. :grinning: